app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,129 @@
+        android:layout_height="0dp"
+        tools:ignore="MissingConstraints"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="@id/vLoginCover"
+        app:layout_constraintStart_toStartOf="@id/vLoginCover"
+        app:layout_constraintEnd_toEndOf="@id/vLoginCover"
+        android:background="@drawable/gradient"
+
+        />
+
+    <ImageView
+        android:id="@+id/vLoginLogo"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:src="@drawable/wolox_logo_large"
+        app:layout_constraintBottom_toTopOf="@id/vEmailInputLogin"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_chainStyle="packed" />
+
+    <EditText
+        android:id="@+id/vEmailInputLogin"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_marginTop="20dp"
+        android:hint="@string/fragment_login_email"
+        android:inputType="textEmailAddress"
+        android:maxLines="1"
+        app:layout_constraintTop_toBottomOf="@id/vLoginLogo"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintBottom_toTopOf="@id/vPasswordInputLogin"
+        />
+
+    <EditText
+        android:id="@+id/vPasswordInputLogin"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="20dp"
+        android:hint="@string/fragment_login_password"
+        android:inputType="textPassword"
+        android:maxLines="1"
+        app:layout_constraintTop_toBottomOf="@+id/vEmailInputLogin"
+        app:layout_constraintStart_toStartOf="@id/vEmailInputLogin"
+        app:layout_constraintEnd_toEndOf="@id/vEmailInputLogin"
+        app:layout_constraintBottom_toTopOf="@id/vButtonLogin"
+        />
+
+    <Button
+        android:id="@+id/vButtonLogin"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="20dp"
+        android:text="@string/fragment_login_button"
+        android:background="@android:color/white"
+        android:shadowColor="@android:color/darker_gray"
+        app:layout_constraintTop_toBottomOf="@+id/vPasswordInputLogin"
+        app:layout_constraintStart_toStartOf="@+id/vPasswordInputLogin"
+        app:layout_constraintEnd_toEndOf="@+id/vPasswordInputLogin"
+        app:layout_constraintBottom_toTopOf="@id/vButtonSignUp"
+        />
+
+    <Button
+        android:id="@+id/vButtonSignUp"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="20dp"
+        android:background="@android:color/holo_blue_light"
```

Comment:
same as above

Reply:
Remember that in rou styleguide, you have all the colors you need for your app. You should add those colors and use those, and not the default android colors. When you call @android:xxx/xxxx you are using always android default resources. It is not bad to use them in some cases, like white color. But this background should be the same as the styleguide